binding/springamqp/src/main/java/io/tracee/binding/springamqp/TraceeMessagePropertiesConverter.java

```diff
@@ -0,0 +1,74 @@
+package io.tracee.binding.springamqp;
+
+import com.rabbitmq.client.AMQP;
+import com.rabbitmq.client.Envelope;
+import io.tracee.Tracee;
+import io.tracee.TraceeBackend;
+import io.tracee.TraceeConstants;
+import io.tracee.configuration.TraceeFilterConfiguration;
+import io.tracee.transport.HttpHeaderTransport;
+import org.springframework.amqp.core.MessageProperties;
+import org.springframework.amqp.rabbit.support.DefaultMessagePropertiesConverter;
+
+import java.util.Arrays;
+import java.util.Map;
+
+import static io.tracee.configuration.TraceeFilterConfiguration.Channel.IncomingResponse;
+import static io.tracee.configuration.TraceeFilterConfiguration.Channel.OutgoingRequest;
+
+public class TraceeMessagePropertiesConverter extends DefaultMessagePropertiesConverter {
+
+	private final TraceeBackend backend;
+	private final HttpHeaderTransport transportSerialization;
```

Comment:
I'm just not sure about storing java maps in headers. We should test it in our Backend, if it works -> why not!